src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -412,6 +413,11 @@
       }
     }
 
+    .pf-m-split-toggle-button {
+      display: flex;
```

Comment:
This is fine, but you can probably add `display: inline-flex; align-items: baseline;` to the base `.pf-[v]-menu-toggle__button` element and wouldn't need it for just this modifier